src/app/helpers/README.md

```diff
@@ -0,0 +1 @@
+Currently we do not differentiate between helpers and utils, put them all here. The only exception is models which have their own directory.
```

Comment:
I think it would be good to explain what falls under `helpers` and what should be put in the `lib` directory instead.

Reply:
I genuinely don't think I can explain constants and helpers/utils any better than the words, perhaps you can suggest something

Reply:
The `lib` directory is where constants needed across multiple components should live. This includes constants relating to styles and assets such as fonts. 

Thinking about this more, it'd make sense to split out helpers into these directories, instead of nesting `test` within `helpers`. 
`testHelpers`
`componentHelpers`
Then I think the directory names make their usage quite clear, for those wanting to know where to put new functions or abstractions.